code/datums/uplink/hardsuit_modules.dm

```diff
@@ -38,3 +38,9 @@
 	name = "Mounted Laser Cannon"
 	item_cost = 16
 	path = /obj/item/rig_module/mounted
+
+/datum/uplink_item/item/tools/rig_cooling_unit
```

Comment:
the path name could use some changes too

Reply:
This is not inside the scope of the pr, rig is still used in many places for the code. Hardsuit should be the in game name.

Reply:
fair